radicle-httpd/src/api/v1/projects.rs

```diff
@@ -378,6 +386,98 @@ async fn issues_handler(
     Ok::<_, Error>(Json(issues))
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+pub struct IssueCreation {
+    pub title: String,
+    pub description: String,
+    pub tags: Vec<Tag>,
+}
+
+// // Create a new issue
+// // `POST /projects/:project/issues`
+async fn issue_creation_handler(
+    Extension(ctx): Extension<Context>,
+    Path(project): Path<Id>,
+    AuthBearer(token): AuthBearer,
+    Json(issue): Json<IssueCreation>,
+) -> impl IntoResponse {
+    let sessions = ctx.sessions.read().await;
+    sessions.get(&token).ok_or(Error::Auth("Unauthorized"))?;
+
+    let storage = &ctx.profile.storage;
+    let signer = ctx.profile.signer().map_err(Error::from)?;
+    let author = signer.public_key();
+    let repo = storage.repository(project)?;
+    let mut issues = Issues::open(*author, &repo)?;
+    issues
+        .create(issue.title, issue.description, &issue.tags, &signer)
+        .map_err(Error::from)?;
+
+    Ok::<_, Error>(())
+}
+
+// // Modifies an issue, defined on `cobs::issue::Action`
+// // `PATCH /projects/:project/issues/:id/:action`
+async fn issue_modify_handler(
+    Extension(ctx): Extension<Context>,
+    Path((project, issue_id, action)): Path<(Id, Oid, String)>,
+    AuthBearer(token): AuthBearer,
+    Json(issue_body): Json<Action>,
+) -> impl IntoResponse {
+    let sessions = ctx.sessions.read().await;
+    sessions.get(&token).ok_or(Error::Auth("Unauthorized"))?;
+
+    let storage = &ctx.profile.storage;
+    let signer = ctx.profile.signer().map_err(Error::from)?;
+    let author = signer.public_key();
+    let repo = storage.repository(project)?;
+    let mut issues = Issues::open(*author, &repo)?;
+    let mut issue = issues.get_mut(&issue_id.into())?;
+
+    match (action.as_str(), issue_body) {
```

Comment:
If we're going this direction, by using the actual CRDT actions, I'd just have an API like this:
```
PATCH         /v1/projects/:project/issues/:id            Action::Assign {...}
```
There's no need to have the action string there. I think in the future we could also directly apply the change to the graph, without calling the individual methods.

Reply:
> we could also directly apply the change to the graph, without calling the individual methods.

Interesting, will look into this now, since it seems to be easier for readability.

Reply:
This has been removed, since I want to focus in this PR on authorize a session.